dataland-backend/src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
 springdoc.show-actuator=true
 springdoc.swagger-ui.oauth2RedirectUrl=https://${PROXY_PRIMARY_URL:-dataland-local.duckdns.org}/api/swagger-ui/oauth2-redirect.html
+springdoc.swagger-ui.oauth.use-pkce-with-authorization-code-grant=true
```

Comment:
Somehow swagger UI authorize doesn't work for me, even though it should by fixed by this.
It still say "invalid redirect_uri"

Reply:
Fixed